docs/source/workingcopy.rst

```diff
@@ -0,0 +1,228 @@
+Working Copy
+============
+
+Plone has the "Working copy" feature provided by the core package ``plone.app.iterate``.
+It allows the user to create a working copy of a (published or live) content object and
+work with it until it's ready to be published without having to edit the original object.
+
+This process has several steps of it's life cycle:
+
+Check-out
+---------
+
+The user initiates the process and creates a "working copy" by "checking out" the content::
+
+POST /plone/document/@workingcopy HTTP/1.1
+Accept: application/json
+Authorization: Basic YWRtaW46c2VjcmV0
+
+and receives the response::
+
+HTTP/1.1 201 Created
+Content-Type: application/json
+Location: http://localhost:55001/plone/working_copy_of_document
+
+Get the working copy
+--------------------
+
+A working copy has been created and can be accessed querying the content::
+
+GET /plone/document/@workingcopy HTTP/1.1
+Accept: application/json
+Authorization: Basic YWRtaW46c2VjcmV0
+
+and receives the response::
+
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "@id": "http://localhost:55001/plone/working_copy_of_document",
+}
+
+the GET content of any object, also states the location of the working copy, if any (``working_copy``).
+
+GET /plone/document HTTP/1.1
```

Comment:
I guess in the future we want to replace this with an auto-generated example (which is not possible right now of course).